phpcs_ruleset.xml

```diff
@@ -3,8 +3,8 @@
     <rule ref="PSR12"/>
     <exclude-pattern>metadata</exclude-pattern>
     <exclude-pattern>src/Google/Ads/GoogleAds/V[0-9]+</exclude-pattern>
-    <exclude-pattern>src/Google/Ads/GoogleAds/Util/Testing/Google</exclude-pattern>
-    <exclude-pattern>src/Google/Ads/GoogleAds/Util/Testing/metadata</exclude-pattern>
+    <exclude-pattern>tests/metadata</exclude-pattern>
+    <exclude-pattern>tests/Google/Ads/GoogleAds/Util/FieldMasks/Proto</exclude-pattern>
```

Comment:
Do you need to account for the files you moved from Lib/Testing?

Reply:
These exclusions only target the generate code. We still want to check the code style of those and I believe it is the case with the current rules.

Reply:
Oops. I misread this as a phpunit file. Sorry.

Reply:
No problem, resolving this comment.